arcgis-ios-sdk-samples/Content Display Logic/ContentPList.plist

```diff
@@ -837,6 +837,14 @@
 				<key>descriptionText</key>
 				<string>This sample demonstrates how to convert a map location Point to a number of different coordinate notations. Tap on the map or edit the values in textfields</string>
 			</dict>
+            <dict>
+                <key>displayName</key>
+                <string>List transformations</string>
+                <key>storyboardName</key>
+                <string>ListTransformations</string>
+                <key>descriptionText</key>
+                <string>This sample demonstrates how to use the TransformationCatalog to get a list of available DatumTransformations that can be used to project a Geometry between two different SpatialReferences, and how to use one of the transformations to perform the GeometryEngine.project operation.</string>
```

Comment:
They were originally class names/methods; there are many differing ways that's handled in the various samples, but the most common seems to avoid actually class names in favor of descriptive words, so instead of "TransformationCatalog", use "transformation catalog".  I've made those changes in this instance.